delivery_mail_templates/__manifest__.py

```diff
@@ -4,7 +4,7 @@
     'name': "Delivery Custom QWeb Email Templates",
```

Comment:
```suggestion
    'name': "Custom Delivery QWeb Email Templates",
```

Reply:
Its better to use more meaningful word first, serves better in search.